BikeRental/Models/Customer.cs

```diff
@@ -4,6 +4,19 @@ namespace BikeRental.Models
 {
     public class Customer : Person
     {
-        public IEnumerable<Reservation> Reservations { get; set; }
+        public Customer()
+        {
+            Reservations = new HashSet<Reservation>();
+        }
+
+        public int CustomerId { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string PhoneNumber { get; set; }
```

Comment:
Can remove CustomerId, FirstName, LastName, PhoneNumber, and Email since Customer already inherits those from Person